ts/components/ui/ItemsPicker.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from "react";
+import { Picker } from "native-base";
+import { StyleSheet } from "react-native";
+import { fromNullable } from "fp-ts/lib/Option";
+import ArrowDown from "../../../img/features/siciliaVola/arrowDown.svg";
+import variables from "../../theme/variables";
+import { isAndroid } from "../../utils/platform";
+import { IOColors } from "../core/variables/IOColors";
+
+type PickerItem = {
+  label: string;
+  value?: string | number;
+};
+
+export type PickerItems = ReadonlyArray<PickerItem>;
+
+type Props = {
```

Comment:
Unless otherwise needed, I recommend using generics to avoid type refinement down the line:

```typescript
type PickerItem<V> = {
  label: string;
  value?: V;
}

type Props<V> = {
    items: ReadonlyArray<PickerItem<V>>;
    onValueChange: (value: V) => void;
    selectedValue?: V;
    //...
}
```

This will:
1. simplify the `value` management within the component
2. simplify the logic in the callbacks and props, since there is only one type at any given time

You can see [on the TS playground](https://www.typescriptlang.org/play?jsx=3#code/JYWwDg9gTgLgBAJQKYEMDG8BmUIjgIilQ3wChRJY4BvOANWCQHc4BfObXAo9GAWgB2KGMABuSMqRgBPMEjgAFYGgDWSKAEkYSEAB46APjgBeGqThwANigBGSSwC44AZxhRgAgOYBuc3FEolgCuSE50vqykUrLyCjhgzvpGptR+FsDaIM5OyCgAJhACltIAglBQKNK6SqrqWjpJBr4WFoV0gSEAwgAWKF6hcAAUAcEDdACUJkaiEMB5zS3O9kgYSHntowD8YQsWAPR7AHTHpJGkB3AAmkhgcBlwGgDKcExIwFB5ADRwNkHwKGAwCgiAIYMULDBuup5MBnHABBAXEFAdAsNA4J4kAJ1MoOEEBBhgIVAnA0LhINjQc5znsXEh5N0YDAEg4Dq50CoIOIoJhLBAmIcySA9ig9gBWADMAE4xQAOACMAHYAGx7eUAJjF6oADJrSGSBK4HplnDU1FATHB9HAkAAPbQCPJwwaudxeAA+AiCIDsUHGBkGYHi2UUwaSk2MRlSLTgRBgQSgAitDGYBmoewAVHBjoc4Bm9qxdHsU0wDKcogajQBZaQAOW9vs65MKWPgpkGEajaSt9SyZvU3ZjGR0zmM1AA2rRrHZHARMO9XPhviMQk55awALqRGMxtodJA9PqYsfDfedmhwTP+fd3OEoeEN9R52msbc7ixLSwrbTrfdj9eDnsZZnJW8A1o8bgeJ4TbgC2oKWh2UxmDGui9qayjmoOLTDlkY6TlYtj2E4+DzlAi7LvuxEtvgm5vu+e6jIe-QniuSDnrQV6sbecD3q6UHPmwdE7p+35rBsIRjvg1FCZewF+EAA) a working example of this proposal.